src/toolkits/supervised_learning/boosted_trees.cpp

```diff
@@ -82,6 +82,12 @@ void init_boosted_trees_common_options(option_manager &options){
       std::numeric_limits<int>::max(),
       false);
 
+  options.create_flexible_type_option(
+      "default_evaluation_metric",
```

Comment:
Why is this called default_evaluation_metric? Can the user change this? Can we still compute multiple metrics?